clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -137,6 +137,11 @@ interface SelectProps {
   isMulti?: boolean;
   variant?: Variant;
   menuPosition?: MenuPosition;
+  /**
+   * If true, when isMulti=false, the selected value will be rendered as a block,
+   * similar to how the multi values are rendered
+   */
+  singleValueBlock?: boolean;
```

Comment:
the difference between 

`singleValueBlock = true`
![image](https://user-images.githubusercontent.com/24641006/220458539-1b3e01b3-efb2-4e15-99e1-bf9811284406.png)

`singleValueBlock = false`
![image](https://user-images.githubusercontent.com/24641006/220458618-ac60f4ca-1a72-406e-b3b2-264286d59102.png)